packages/@react-spectrum/actionbar/docs/ActionBar.mdx

```diff
@@ -0,0 +1,240 @@
+{/* Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import Anatomy from './ActionBarAnatomy.svg';
+import docs from 'docs:@react-spectrum/actionbar';
+import {HeaderInfo, PropTable, PageDescription, TypeLink} from '@react-spectrum/docs';
+import packageData from '@react-spectrum/actionbar/package.json';
+
+```jsx import
+import {ActionBar, ActionBarContainer, Item} from '@react-spectrum/actionbar';
+```
+
+---
+category: Collections
+keywords: [actionbar, selection, actions]
+after_version: 3.0.0-alpha.4
+---
+
+# ActionBar
+
+<PageDescription>{docs.exports.ActionBar.description}</PageDescription>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['ActionBar, ActionBarContainer, Item']}
+ />
+
+## Example
+```tsx example
+import {Cell, Column, Row, TableBody, TableHeader, TableView} from '@react-spectrum/table';
+import {Text} from '@react-spectrum/text';
+
+function StaticActionBar() {
```

Comment:
This example takes a lot of vertical space unfortunately. What do people think about me reducing this and the dynamic example to 
```
    <ActionBarContainer>
      <>
      <ActionBar
        isEmphasized
        selectedItemCount={selectedKeys === 'all' ? selectedKeys : selectedKeys.size}
        onClearSelection={() => {
          setSelectedKeys(new Set());
        }}>
        <Item key="edit">
          Edit
        </Item>
        <Item key="copy">
          Copy
        </Item>
        <Item key="delete">
          Delete
        </Item>
      </ActionBar>
    </ActionBarContainer>
```
so that it only shows the ActionBarContainer? Then the example in the `Events` section I can include the TableView and it won't be the same example basically used 4 times...

Reply:
Updated the examples, people can compare and contrast the two builds